src/completers/git.zsh

```diff
@@ -511,6 +511,114 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = 'show' ]]; then
+        local prefix_option completing_option
+        local git_options_argument_required=(
+            -l -G -O -S -U
+            --anchored --color-moved-ws --diff-algorithm --diff-filter --dst-prefix
+            --encoding --expand-tabs --find-object --format --inter-hunk-context
+            --line-prefix --output --output-indicator-context --output-indicator-new
+            --output-indicator-old --src-prefix --unified --word-diff-regex --ws-error-highlight
+        )
+        local git_options_argument_optional=(
+            -B -C -M -X
+            --abbrev --break-rewrites --color --color-moved --dirstat
+            --find-copies --find-renames --ignore-submodules --notes
+            --pretty --relative --show-notes --stat --submodule --word-diff
+        )
```

Comment:
No need to break lines. Either write in a single line or write every item in different lines.